analyzer/src/funTest/assets/projects/synthetic/pip-expected-output.yml

```diff
@@ -14,21 +14,15 @@ project:
     path: ""
   vcs_processed:
     type: "Git"
-    url: "<REPLACE_URL>"
-    revision: "<REPLACE_REVISION>"
-    path: "<REPLACE_PATH>"
+    url: "ssh://git@github.com/heremaps/oss-review-toolkit.git"
+    revision: "d9fdc9b6a9885a5ebd8a4e2fb19565c05ee31d14"
+    path: "analyzer/src/funTest/assets/projects/synthetic/pip"
   homepage_url: "https://example.org/app"
   scopes:
   - name: "install"
     dependencies:
     - id: "PyPI::Flask:1.0"
-      dependencies:
-      - id: "PyPI::Jinja2:2.10.1"
-        dependencies:
-        - id: "PyPI::MarkupSafe:1.0"
-      - id: "PyPI::Werkzeug:0.14.1"
-      - id: "PyPI::click:6.7"
-      - id: "PyPI::itsdangerous:0.24"
+    - id: "PyPI::pkg-resources:0.0.0"
```

Comment:
I thought this should not be listed according to https://github.com/heremaps/oss-review-toolkit/pull/1702...